src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -38,10 +38,15 @@
     private static final String FIND_TRANSACTIONS_BY_PARENT_EXT_ID = "SELECT * FROM transaction " +
             " WHERE parent_external_id = :parentExternalId";
 
-    private static final String SEARCH_TRANSACTIONS = "SELECT * FROM transaction t " +
+    private static final String SEARCH_TRANSACTIONS = "SELECT t.*, po.paid_out_date AS paid_out_date FROM transaction t " +
+            "LEFT OUTER JOIN payout po on " +
+            "t.gateway_payout_id = po.gateway_payout_id " +
+            ":payoutAccountId " +
             ":searchExtraFields " +
             "ORDER BY t.created_date DESC OFFSET :offset LIMIT :limit";
 
+    private static final String SEARCH_TRANSACTIONS_WITH_PAYOUT = "AND po.gateway_account_id IN (<account_id>) ";
```

Comment:
(Small one) Could it be renamed to indicate it is a clause or inline this as it is not full search query ?